### PR TITLE
Fixes #15126: Add verbosity level in the rudder logger

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15126

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

File: tree/20_cfe_basics/log_rudder.cf [hunk 1]
```diff
@@ -16,9 +16,19 @@
...
+# @description Logging different verbosity outputs for Rudder reports
+# Supported mode are:
+#  log_repaired
+#  log_warn
+#  log_info
+#  log_debug
+#  log_trace
 #
+# If the parameter mode is not in the list above, it will report in a classical manner
+# based on the reporting context and the defined classes.
```
Comment (ncharles) [inline tree/20_cfe_basics/log_rudder.cf]: I'm really confused on how it should work: it has a class prefix, that it won't use to get the _repaired, _success, _failure, so the message will also be not be customized

why not simply having a 
bundle agent log_rudder_mode(mode, message, class_parameter, class_prefix) ?
signature will only expose what's relevant, and make it easier to use.

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**